python/src/scipp/plotting/figure.py

```diff
@@ -53,6 +53,22 @@ def __init__(self,
         self.axlocator = {}
         self.xlabel = xlabel
         self.ylabel = ylabel
+        self.axis_offsets = {
+            "x":
+            self.ax.text(0.70,
+                         -0.07,
+                         "",
+                         ha='left',
+                         va='top',
+                         transform=self.ax.transAxes),
+            "y":
+            self.ax.text(0.0,
+                         1.0,
```

Comment:
What do these magic numbers mean?

Reply:
they are x and y coordinates in canvas units that range from 0 to 1
will add comment also